_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why is it so important to do pre-GSoC analysis and
+ design as early as possible, to be finalized in the cEP?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the coala community to
+discuss the new major features that the contributor intends to implement during
+GSoC.
+
+**cEPs** are designed before implementation. Implementing major features
+usually happens from a technical perspective and may lead to poorly designed
+content - developer and user wise. Designing cEPs can essentially save a lot of
+coding work and also makes sure that major features are properly documented.
+
+Experience tells us that the best GSoC students are the ones who heavily
+contribute to the project ideas website, create projects from the issues, and
```

Comment:
Not `contributed` instead?

Reply:
`ideas'`?

Reply:
I agree with frextrite, chk [here](
https://github.com/coala/projects/pull/693#discussion_r242912841)